src/power_grid_model/validation/errors.py

```diff
@@ -237,9 +237,9 @@ class InvalidEnumValueError(SingleFieldValidationError):
     """
 
     _message = "Field {field} contains invalid {enum} values for {n} {objects}."
-    enum: Type[Enum]
+    enum: Union[Type[Enum], List[Type[Enum]]]
```

Comment:
Wait a minute, what did I just see?

Reply:
Oh, never mind, this is python code

Reply:
🙃

Reply:
when we drop support for python 3.9 we can officially move to using the newer `A | B` syntax instead of `Union[A, B]`. Some ongoing discussions even propose deprecating `Union` (which of course would be a breaking change but ... that's not for us to decide)